core/src/test/scala/org/broadinstitute/dsde/rawls/monitor/V1WorkspaceMigrationMonitorSpec.scala

```diff
@@ -39,4 +47,39 @@ class V1WorkspaceMigrationMonitorSpec
       }
     }
   }
+
+  // use an existing test project (broad-dsde-dev)
+  "createTempBucket" should "create a new bucket in the same region" ignore {
```

Comment:
temporary "integration unit test" for verifying our code while it is in progress. Marked with "ignore" since it won't run on CI.